src/components/media-card/media-card.stories.js

```diff
@@ -0,0 +1,50 @@
+import '../../styles/theme.css';
+import './media-card.js';
+
+const MOCK_IMG_ASSETS = JSON.stringify({
+  alt: 'test image',
+  format: 'jpg',
+  height: '400',
+  width: '600',
+  publicId: 'test-image-id',
+  resourceType: 'image'
+});
+
+const MOCK_VIDEO_ASSETS = JSON.stringify({
+  alt: 'test video',
+  format: 'mp4',
+  height: '400',
+  width: '600',
+  publicId: 'example-video-id',
+  resourceType: 'video'
+});
+
+export default {
+  title: 'Components/Media Card'
+};
+
+const Template = ({ props }) => {
+  const { assest } = props;
+  return `
+    <bf-media-card
+     assest="${assest}"
```

Comment:
Looks like there was a small typo here on `assest` but with that little tweak and using some real `publicId`s this now looks good!
![Screenshot 2024-05-25 at 2 55 00 PM](https://github.com/AnalogStudiosRI/www.blissri.com/assets/895923/67666bfa-8bd8-47fa-8823-dfd84ebf9aee)
![Screenshot 2024-05-25 at 2 55 04 PM](https://github.com/AnalogStudiosRI/www.blissri.com/assets/895923/4b050e31-451b-4700-9c58-62e3c16d1c79)

Reply:
Ohh good catch!